src/routes/write-exercise/components/PopoverContent.tsx

```diff
@@ -1,35 +1,36 @@
-import React, { ReactElement } from 'react'
-import { View, Text, StyleSheet } from 'react-native'
+import React from 'react'
 import { InfoIcon } from '../../../../assets/images'
 import { widthPercentageToDP as wp } from 'react-native-responsive-screen'
 import { COLORS } from '../../../constants/colors'
 import labels from '../../../constants/labels.json'
+import styled from 'styled-components/native'
 
-export const styles = StyleSheet.create({
-  container: {
-    flexDirection: 'row',
-    alignItems: 'center',
-    backgroundColor: COLORS.lunesBlack,
-    width: wp('80%'),
-    height: 60,
-    paddingVertical: 8,
-    paddingHorizontal: 9,
-    borderRadius: 2
-  },
-  message: {
-    color: COLORS.lunesWhite,
-    fontSize: wp('3.5%'),
-    fontWeight: 'normal',
-    fontFamily: 'SourceSansPro-Regular',
-    width: wp('60%'),
-    marginLeft: 8
-  }
-})
-const PopoverContent = (): ReactElement => (
-  <View style={styles.container}>
+const StyledComponent = styled.View`
+  flex-direction: row;
+  align-items: center;
+  background-color: ${COLORS.lunesBlack};
+  width: ${wp('80%')};
+  height: 60;
+  padding-right: 8;
+  padding-left: 8;
+  padding-top: 9;
+  padding-right: 9;
+  border-radius: 2;
```

Comment:
you should always add a unit for paddings, margins, width, etc. otherwise there will be tons of styled component warnings when running the app. Generally just add px. You can only omit the unit for the shadow attributes, elevation and when the value is 0
```suggestion
  background-color: ${COLORS.lunesBlack};
  width: ${wp('80%')}px;
  height: 60px;
  padding-right: 8px;
  padding-left: 8px;
  padding-top: 9px;
  padding-right: 9px;
  border-radius: 2px;
```